src/main/java/com/redhat/cloud/policies/app/auth/RhIdPrincipal.java

```diff
@@ -25,20 +25,18 @@ public class RhIdPrincipal implements Principal {
 
     private String name;
     private String account;
+    private String orgId;
     private boolean canReadPolicies;
     private boolean canWritePolicies;
     private String rawRhId;
 
-    public RhIdPrincipal() {
-    }
```

Comment:
This could be needed by some code building an instance from reflection code. Or not...